pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -731,24 +732,29 @@ func handleCreate(ctx *verifierContext, objType string,
 		State:       types.DOWNLOADED,
 		RefCount:    config.RefCount,
 		LastUse:     time.Now(),
+		IsContainer: config.IsContainer,
 	}
 	publishVerifyImageStatus(ctx, &status)
 
-	ok, size := markObjectAsVerifying(ctx, config, &status)
-	if !ok {
-		log.Errorf("handleCreate fail for %s\n", config.Name)
-		return
-	}
-	status.Size = size
-	publishVerifyImageStatus(ctx, &status)
+	// Skip verification steps for Container. Container image verification
+	// is done by rkt fetch itself.. No need to be done here.
```

Comment:
Actually for the purposes of defense in depth it should be done in the verifier and not in the downloader.

Reply:
Not sure I understand - rkt fetch also verifies the image. So why is there a need for another verification step?? 
https://coreos.com/rkt/docs/latest/signing-and-verification-guide.html

Currently, we are passing --insecure-options=image. If we don't pass it, fetch will verify the image too. So, "rkt fetch = download + verify" .